### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by OmarMWarraich**
### In this PR, following project requirement has been completed(Screenshot appended):

- [ ] Use primary key & foreign key mechanism for joining tables,
- [ ] Query multiple tables.
- [ ] Prepare complex Queries that answer analytical questions.

Screenshot from 2022-11-12 14-39-55

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -12,3 +12,36 @@ CREATE TABLE animals (
 /* Add a new column species of type string to the animals table. */
 
 ALTER TABLE animals ADD species varchar(100);
+
+/* Create a table named owners with the following columns: */
+
+CREATE TABLE owners (
+    id SERIAL NOT NULL PRIMARY KEY,
```
Comment (cynthiainga) [inline schema.sql]: - [ ] The `id` should be the integer, p[lease kindly fix that 🙃

File: schema.sql [hunk 2]
```diff
@@ -12,3 +12,36 @@ CREATE TABLE animals (
...
+    full_name varchar(100),
+    age int,
+);
+
+/* Create a table namex species with the following columns: */
+
+CREATE TABLE species (
+    id SERIAL NOT NULL PRIMARY KEY,
```
Comment (cynthiainga) [inline schema.sql]: - [ ] The id should be the integer, p[lease kindly fix that 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @OmarMWarraich, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @OmarMWarraich.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- You have implemented all suggestions from the previous review 👍 

### Optional suggestions

- Please, consider adding the license for your repo - it will look more professional 👍 Here is [the instruction](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository), which can help 💯 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**